apps/admin-gui/src/app/admin/pages/admin-page/admin-services/service-detail-page/service-detail-page.component.html

```diff
@@ -35,6 +35,25 @@
       <span class="mt-1 entity-info">
         {{'SERVICE_DETAIL.DESCRIPTION' | translate}}: {{service.description}}
       </span>
+      <div class="mt-1 entity-info">
+        {{'SERVICE_DETAIL.STATUS' | translate}}:
+        <span *ngIf="service.enabled">
+          {{'SERVICE_DETAIL.ENABLED' | translate}}
+          <button mat-icon-button (click)="changeServiceStatus()">
+            <mat-icon class="align-content-center">
+              check_circle_outline
+            </mat-icon>
+          </button>
+        </span>
+        <span *ngIf="!service.enabled">
+          {{'SERVICE_DETAIL.DISABLED' | translate}}
+          <button mat-icon-button (click)="changeServiceStatus()">
+            <mat-icon class="align-content-center">
+              block
+            </mat-icon>
+          </button>
+        </span>
+      </div>
```

Comment:
I think that the html can be simplified for better readability, but it's not necessary. Otherwise it's good. 
```suggestion
      <div class="mt-1 entity-info">
        {{'SERVICE_DETAIL.STATUS' | translate}}:
        <span>
          {{service.enabled ? ('SERVICE_DETAIL.ENABLED' | translate) : ('SERVICE_DETAIL.DISABLED' | translate)}}
          <button mat-icon-button (click)="changeServiceStatus()">
            <mat-icon class="align-content-center">
              {{service.enabled ? "check_circle_outline" : "block"}}
            </mat-icon>
          </button>
        </span>
      </div>
```